pkg/provider/cloud/vsphere/ca_test.go

```diff
@@ -28,6 +28,7 @@ import (
 )
 
 func TestVSphereCA(t *testing.T) {
+	t.Skip()
```

Comment:
Please leave a comment as to why this test is skipped. I predict that this test would otherwise stay skipped forever and ever after.

Reply:
Yes a comment would be good for explaining why. However it would not help in piking this up again. Therefore I am going to link this to the corresponding issue that should enable the test again once done.